specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/disasterRecoveryConfigurations.json

```diff
@@ -0,0 +1,405 @@
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/disasterRecoveryConfiguration/{disasterRecoveryConfigurationName}": {
+      "delete": {
+        "tags": [
+          "DisasterRecoveryConfigurations"
+        ],
+        "operationId": "DisasterRecoveryConfigurations_Delete",
+        "description": "Deletes a disaster recovery configuration.",
+        "x-ms-examples": {
+          "Delete a disaster recovery configuration": { "$ref": "./examples/DisasterRecoveryConfigurationDelete.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "disasterRecoveryConfigurationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the disaster recovery configuration to be deleted."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "NoContent"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "DisasterRecoveryConfigurations"
+        ],
+        "operationId": "DisasterRecoveryConfigurations_CreateOrUpdate",
+        "description": "Creates or updates a disaster recovery configuration.",
+        "x-ms-examples": {
+          "Update a disaster recovery configuration": { "$ref": "./examples/DisasterRecoveryConfigurationCreateOrUpdate.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "disasterRecoveryConfigurationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the disaster recovery configuration to be created/updated."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/DisasterRecoveryConfiguration"
+            }
+          },
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DisasterRecoveryConfiguration"
+            }
+          }
+        }
```

Comment:
Given op returns `201` & `200` it seems it is LRO, should we mark it so?

Reply:
Also fixed

Reply:
also 202